akka-actor/src/main/resources/reference.conf

```diff
@@ -420,6 +420,7 @@ akka {
         mailbox-type = "akka.dispatch.UnboundedMailbox"
       }
 
+      # deprecated (because relies on blocking)
```

Comment:
My mistake, all bounded queues have a timeout (timeout handling added via a trait that I missed to look into). But if it's not those we want to deprecate, I don't know which ones.